public/app/features/explore/ExplorePanel.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { AbsoluteTimeRange, DataFrame, dateTime, LoadingState } from '@grafana/data';
+import { PanelRenderer } from '@grafana/runtime';
+import { PanelChrome } from '@grafana/ui';
+
+export interface Props {
+  width: number;
+  height: number;
+  timeZone: string;
+  frame: DataFrame;
+  absoluteRange: AbsoluteTimeRange;
+  state: LoadingState;
+}
+
+export function ExplorePanel({ width, height, timeZone, state, frame, absoluteRange }: Props) {
```

Comment:
Wondering what you think about naming this something like `CustomPanel` so it doesn't sound so generic.

Reply:
Seems to be a bit of a mix, I will rename it to `CustomContainer` as it seem to fit best.

Reply:
I have now renamed the container.